toyota_iQ_2009_can.dbc

```diff
@@ -0,0 +1,432 @@
+VERSION ""
+
+
+NS_ :
+    NS_DESC_
+    CM_
+    BA_DEF_
+    BA_
+    VAL_
+    CAT_DEF_
+    CAT_
+    FILTER
+    BA_DEF_DEF_
+    EV_DATA_
+    ENVVAR_DATA_
+    SGTYPE_
+    SGTYPE_VAL_
+    BA_DEF_SGTYPE_
+    BA_SGTYPE_
+    SIG_TYPE_REF_
+    VAL_TABLE_
+    SIG_GROUP_
+    SIG_VALTYPE_
+    SIGTYPE_VALTYPE_
+    BO_TX_BU_
+    BA_DEF_REL_
+    BA_REL_
+    BA_DEF_DEF_REL_
+    BU_SG_REL_
+    BU_EV_REL_
+    BU_BO_REL_
+    SG_MUL_VAL_
+
+BS_:
+
+BU_: XXX
+
+
+BO_ 528 TRACK_A_0: 8 XXX
```

Comment:
@alessbelli , can you remove all those radar signals? basically from here to line 256. Since this is a new dbc file, you should only report the messages and signals that you were able to reverse engineer.

Reply:
You're entirely right, I think I started with a toyota base, but I forgot to remove non-matching signals since they did not appear in cabana. Cleaning it right away